code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -18,9 +18,10 @@
 	var/list/data = null
 	var/volume = 0
 	var/nutriment_factor = 0
-	var/custom_metabolism = REAGENTS_METABOLISM
-	var/overdose = 0
-	var/overdose_dam = 1
+	var/metabolization_rate = REAGENTS_METABOLISM
```

Comment:
I hung onto this because I've got chems with different metabolization rates being added.

I absolutely hate the 0.01 metab rate chems and I want to remove them but that's out of the scope of this PR.